index.js

```diff
@@ -0,0 +1,83 @@
+let middleDiv = document.querySelector("#middleDiv");
```

Comment:
Very well done in terms of styling and functionality. I left some feedbacks below but they are mostly minor. I can see that you have good understanding of DOM manipulation as well as utilizing data fetched from API servers. Keep up the good work!